src/views/preview/preview.jsx

```diff
@@ -165,8 +166,11 @@ class Preview extends React.Component {
                 });
             });
     }
-    handleDeleteComment (id) {
-        this.props.handleDeleteComment(this.state.projectId, id, this.props.user.token);
+    handleAddComment (comment, topLevelCommentId) {
+        this.props.handleAddComment(comment, topLevelCommentId);
+    }
+    handleDeleteComment (id, topLevelCommentId) {
+        this.props.handleDeleteComment(this.state.projectId, id, topLevelCommentId, this.props.user.token);
```

Comment:
Seeing these handlers points out that we have an odd convention here of passing `handleX` as a prop. Usually we use `onX` for props, `handleX` for the internal handler.